<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>939</width>
    <height>481</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>微软雅黑</family>
    <pointsize>13</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QComboBox" name="cBoxPort">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>黑体</family>
      <pointsize>13</pointsize>
     </font>
    </property>
    <item>
     <property name="text">
      <string>COM0</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>COM1</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>COM2</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>COM3</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>COM4</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>COM5</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>COM6</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>COM7</string>
     </property>
    </item>
   </widget>
   <widget class="QComboBox" name="cBoxBaud">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>10</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>黑体</family>
      <pointsize>13</pointsize>
     </font>
    </property>
    <item>
     <property name="text">
      <string>1200</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>2400</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>4800</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>9600</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>38400</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>57600</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>115200</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>COM7</string>
     </property>
    </item>
   </widget>
   <widget class="QComboBox" name="cBoxDataBits">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>10</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>黑体</family>
      <pointsize>13</pointsize>
     </font>
    </property>
    <item>
     <property name="text">
      <string>6</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>7</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>8</string>
     </property>
    </item>
   </widget>
   <widget class="QComboBox" name="cBoxParity">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>10</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>黑体</family>
      <pointsize>13</pointsize>
     </font>
    </property>
    <item>
     <property name="text">
      <string>N</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>E</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>O</string>
     </property>
    </item>
   </widget>
   <widget class="QComboBox" name="cBoxStop">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>10</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>黑体</family>
      <pointsize>13</pointsize>
     </font>
    </property>
    <item>
     <property name="text">
      <string>1</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>2</string>
     </property>
    </item>
   </widget>
   <widget class="QPushButton" name="btnOnOff">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>10</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>打开</string>
    </property>
   </widget>
   <widget class="QLabel" name="labLight">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>10</y>
      <width>30</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>light</string>
    </property>
   </widget>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>939</width>
     <height>29</height>
    </rect>
   </property>
  </widget>
  <action name="actionD">
   <property name="text">
    <string>d </string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
